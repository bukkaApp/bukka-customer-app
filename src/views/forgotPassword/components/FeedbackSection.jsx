import React from 'react';

import PropTypes from 'prop-types';

import EmailSent from 'Components/icons/EmailSent';
import Container from 'Components/container';
import GoToLogin from '../common/GoToLogin';

import './feedbacksection.scss';

const Feedback = ({ push }) => (
  <>
    <Container classNames="pt-5 pb-2 px-4">
      <div className="col-3 mx-auto">
        <EmailSent />
      </div>
      <div className="py-2">
        <h3 className="ft-18">
          Instructions on resetting your password are on their way!
        </h3>
        <br />
        <h3 className="ft-15">
          If you {"don't"} receive an email soon ensure that the email is valid
          <i className="text-muted"> (retrieve your email(s))</i> and that you
          can receive emails from support@mybukka.com.
        </h3>
        <h3 className="ft-15">
          Contact support@mybukka.com for additional assistance.
        </h3>
      </div>
    </Container>
    <div className="pb-4">
      <GoToLogin push={push} />
    </div>
  </>
);

export default Feedback;

Feedback.defaultProps = {
  push: () => {}
};

Feedback.propTypes = {
  push: PropTypes.func
};
