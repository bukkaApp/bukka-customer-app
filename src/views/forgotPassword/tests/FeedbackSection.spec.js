import React from 'react';
import FeedbackSection from '../components/FeedbackSection';


describe('Input field component', () => {
  const props = {
    push: jest.fn()
  };
  const wrapper = shallow(<FeedbackSection {...props} />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
