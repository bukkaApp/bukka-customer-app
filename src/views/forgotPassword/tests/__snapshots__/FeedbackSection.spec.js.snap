// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input field component renders properly 1`] = `ShallowWrapper {}`;
